src/codemap_details/Components/Query.js

```diff
@@ -1,6 +1,56 @@
 import React from "react";
 import styles from "../CSS/Query.module.css";
+import query from "../assets/query.png";
+import { Jumbotron, Col, Row, Image, Container } from "react-bootstrap";
 
 export default function Query() {
-  return <React.Fragment />;
+  return (
+    <div>
+      <Jumbotron className={styles.jumbo} fluid>
+        <Container className={styles.container}>
+          <h1 className={styles.head}>
+            What if you have a <span className={styles.query}>Query</span>?
+          </h1>
+          <Row>
+            <Col lg={6}>
+              <p className={styles.para}>
+                Duis aute irure dolor in reprehenderit in voluptate velit esse
+                cillum dolore eu fugiat nulla pariatur.
+                <br />
+                <br />
+                Excepteur sint occaecat cupidatat non proident, sunt in culpa
+                qui officia deserunt mollit anim id est laborum.
+              </p>
+              <hr className={styles.line} />
+            </Col>
+            <Col lg={6}>
+              <Image src={query} className={styles.img} fluid />
+            </Col>
+          </Row>
+        </Container>
+        <Container className={styles.small} fluid>
+          <h1 className={styles.head} style={{ textAlign: "center" }}>
+            What if you have a <span className={styles.query}>Query</span>?
+          </h1>
+          <Image
+            className={styles.img}
+            style={{ maxWidth: "90%" }}
+            src={query}
+            fluid
+          />
+          <p className={styles.para} fluid>
+            Duis aute irure dolor in reprehenderit in voluptate velit esse
+            cillum dolore eu fugiat nulla pariatur.
+            <br />
```

Comment:
@prachibansal2000 Why are we using multiple `<br />` tags? Can't we have `margin-bottom`?

Reply:
i have added `<br/>` tags in between the text.

Reply:
Better to use `<p>` tags imo

Reply:
@prachibansal2000 What about `<span>` in between text? And span will have `margin-bottom`?

<p> tag adds a single line break by default but I can see you've put twice.

Reply:
i have removed all the `<br/>` tags